### PR TITLE
Add H5P components

### LINKED ISSUE #126: Add H5P components

### PR DESCRIPTION
**Pull request by dialexo**
Closes #126 

Adds H5P icon and standalone player

### CODE REVIEW

**Review by spaenleh — APPROVED** [review]
Looks Good ! 🥁5️⃣

**Review by pyphilia — APPROVED** [review]
💯

File: src/types/h5p/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  /** Used for xAPI events */
+  class XAPIEvent extends Event {
+    /** List of verbs defined at ADL xAPI Vocabulary */
+    static allowedXAPIVerbs: Array<string>;
+    /** Get content xAPI ID */
+    getContentXAPIId(instance: H5P.Instance);
+    /** Get the max value of the result - score part of the statement */
+    getMaxScore(): number | null;
+    /** Get the raw valu of the result - score part of the statement */
+    getScore(): number | null;
```
Comment (juancarlosfarah) [inline src/types/h5p/index.d.ts:45]: What are these scores?

File: src/types/h5p/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    /**
+     * Helper function to create xAPI completed events
+     * @deprecated
+     * @use triggerXAPIScored instead
+     */
+    triggerXAPICompleted(score: number, maxScore: number, success: boolean);
+    /** Helepr function to create scored xAPI events */
+    triggerXAPIScored(
+      score: number,
+      maxScore: number,
```
Comment (juancarlosfarah) [inline src/types/h5p/index.d.ts:97]: Same as above. Not sure what these mean.

File: src/types/h5p/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    );
+    /** Helper function for triggering xAPI added to the EventDispatcher */
+    triggerXAPI(verb: string, extra?: object);
+    /**
+     * Helper function to create xAPI completed events
+     * @deprecated
+     * @use triggerXAPIScored instead
+     */
+    triggerXAPICompleted(score: number, maxScore: number, success: boolean);
+    /** Helepr function to create scored xAPI events */
```
Comment (juancarlosfarah) [inline src/types/h5p/index.d.ts:94]: Typo: Helper

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good to me! Just some minor comments and questions.

File: src/types/h5p/index.d.ts (hunk 1 above)
Comment (dialexo) [reply src/types/h5p/index.d.ts:45]: They come from the H5P API reference at https://h5p.org/documentation/api/H5P.html
I've only added the basic types from the global H5P namespaces for Typescript.

File: src/types/h5p/index.d.ts (hunk 2 above)
Comment (dialexo) [reply src/types/h5p/index.d.ts:97]: They come from the H5P API reference at https://h5p.org/documentation/api/H5P.html
I've only added the basic types from the global H5P namespaces for Typescript.

**Pull request merged**